studio/LonaStudio/Plugins/LonaPlugins.swift

```diff
@@ -46,11 +46,14 @@ class LonaPlugins {
         func run(onSuccess: (String) -> Void) {
             guard let config = config else { return }
 
-            LonaNode.run(
+            let rpcService = RPCService()
+
+            let stdinFileHandle = LonaNode.run(
```

Comment:
I think we should expose an RPC interface of some kind instead of a file handle. This would be OK on methods only used internally in this file, but for methods we call in other parts of LS a file handle should be an implementation detail (e.g. maybe the call actually goes over HTTP/websockets).

Reply:
Thinking out loud, maybe internally in the file we pass around a `Process` instead of file handles

Reply:
So it doesn't prevent the node process from termination if the node process is not listening to stdin. It's only when the node process listens to it to it won't terminate (which is expected) and that's why there is an explicit `process.exit(0)` at the end of the plugin